docs/en/stack/data-frames/limitations.asciidoc

```diff
@@ -0,0 +1,54 @@
+[[dataframe-limitations]]
+== {dataframe-cap} limitations
+
+beta[]
+
+The following limitations and known problems apply to the 7.2 release of 
+the Elastic {dataframe} feature:
+
+[float]
+[[df-wizard-limitations]]
+=== {dataframe-cap} wizard limitation
+
+If you have fields that contain arrays, you cannot select these fields in the source 
+index table in the {kib} {dataframe} wizard. You can, however, use those fields 
+when you group and aggregate your data in the wizard.
+
+[float]
+[[df-ccs-limitations]]
+=== {ccs-cap} limitation
+
+{ccs-cap} is not supported in 7.2 for {dataframe-transforms}.
```

Comment:
This is by design ... not sure if we need this documented as a limitations, as it should be documented against the `transform_id`.

- only allow lowercase alphanumeric (a-z and 0-9), hyphens or underscores
- must start and end with alphanumeric
- maximum length of 64 characters